plugin/type_inet/sql_type_inet.cc

```diff
@@ -493,8 +493,13 @@ size_t Inet6::to_string(char *dst, size_t dstsize) const
       //
       // If it is not the last field, append closing ':'.
 
-      p += sprintf(p, "%x", ipv6_words[i]);
+      int ret= snprintf(p, dstsize_available, "%x", ipv6_words[i]);
```

Comment:
With Vista EOL this could be `inet_ntop` now. Btw good pattern, a recent security conference shows the dangers of `p+= snprintf(p` because it doesn't mean that the full length was written to the buffer.

Is it really safe to return without a SQL warning pushed?

Reply:
The goal of this task is to modify the server source code to make it compilable. That is, replace sprintf to snprintf at every place where building of core server without extra plugin (e.g. CONNECT engine) fails. Any other modifications is out of scope. If such modifications are absolutely required they should be filed in JIRA as separate tasks.